docs/cases/api/actions-api/cases-actions-api-intro.asciidoc

```diff
@@ -25,11 +25,7 @@ To send cases to an external system and keep the {es-sec-ui} updated:
 . <<assign-connector>> or <<case-api-update-connector>>: If required, configure
 connector options (Cases API).
 . [[sn-returned-data]]<<cases-actions-api-execute>>: Send the case to an
-external system (Actions API). You must store the returned data as it is
```

Comment:
@cnasikas I'm not sure if we should put something else in this step since I don't believe you can _Update the case with the associated external system data return_ any longer (the API doesn't exist).

Reply:
You are right about deleting step 4. I think the steps are a bit wrong. There is no need for the second step. You can push a case without a configuration. I think it should be:

1. Create a connector
2. Create a case with the connector created in step 1.
3. Push the case.

What do you think @nastasha-solomon ?